corehq/apps/reports/standard/deployments.py

```diff
@@ -165,18 +165,20 @@ def rows(self):
 
         user_id = self.request.GET.get('individual')
         if not user_id:
-            return []
-
-        # security check
-        get_document_or_404(CommCareUser, self.domain, user_id)
-
+            key_args = {}
```

Comment:
(I'm also happy to take this if you want since it's sort of a QA bug on my feature)